app/templates/bulk_resource/review_desc_options.html

```diff
@@ -31,15 +31,19 @@
 
         <div class="twelve wide column">
             <h2 class="ui header">
-                Review
+                CSV Upload Workflow
             </h2>
         </div>
         <div class="twelve wide column">
-            For each of the columns that you marked as options, we have
-            identified the options already present in your CSV file. If
-            any of the existing options are incorrect, please edit the original
-            CSV file and re-upload it. We will store the options in alphabetical
-            order.
+          <h4 class="ui header">
+            Review Option Descriptor Option Values
+            <div class="sub header">
+              For each of the column headers that you marked as option descriptors,
+              we have identified the option values present in your CSV file. If
```

Comment:
can we write that the CSV adds to these options? at first read, the language implies that we're overwriting them with CSV. Also write that you can add option values in-app.